etc/config_test.go

```diff
@@ -0,0 +1,24 @@
+package etc
+
+import (
+	"github.com/stretchr/testify/suite"
```

Comment:
please sort imports

The `goimports` command can do this for you automatically.